ui/next/app/redux/uiData.ts

```diff
@@ -153,16 +153,22 @@ export function loadUIData(...keys: string[]) {
     dispatch(fetchUIData());
 
     return getUIData({ keys }).then((response) => {
-      response.getKeyValues().forEach((val, key) => {
+      let keyValues = response.getKeyValues();
+
+      _.each(keys, (key) => {
```

Comment:
`keys` is actually the original list of keys to get, passed in on line 151. I switched to iterating through those keys instead so I could distinguish between "fetched but missing" and "not yet fetched" depending on if the key is present.